### PR TITLE
Setup docker container for those that wish to use it

### PR DESCRIPTION
**Pull request by jrgifford**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 default: &default
   adapter: postgresql
   encoding: utf8
-  host: localhost
-  username:
-  password:
+  host: <%= ENV.fetch('DB_HOST', 'localhost') %>
+  username: <%= ENV.fetch('POSTGRES_USER', 'twentyliters') %>
+  password: <%= ENV.fetch('POSTGRES_PASSWORD', 'password3214') %>
```
Comment (Ross-Hunter) [inline config/database.yml]: Remove these defaults? I'd rather get an error message about not setting this variable than a weird message about this username and password.

Comment (jrgifford) [reply config/database.yml]: I could go either way on this - I just didn't want to break this for anyone who isn't using docker.

Comment (jrgifford) [reply config/database.yml]: Didn't realize we had dotenv-rails in the project, I'll go ahead and remove the defaults.

Comment (Ross-Hunter) [reply config/database.yml]: I'm down with the fetch, but just not the 'twentyliters' and 'password3214'. I'm fine with forcing folks will need to set up a `.env` file.

Comment (jrgifford) [reply config/database.yml]: Removed the defaults and included an example .env file in this PR.

**Comment by Ross-Hunter** [comment]
👍

**Pull request merged**